src/routes/docs/tooling/command-line/non-interactive/+page.markdoc

```diff
@@ -19,39 +19,55 @@ In this mode, the CLI can only interact with one project at a time.
 # API Keys {% #api-keys %}
 In non-interactive mode, the CLI uses an API key to authenticate. Your API key must have sufficient permissions to execute the commands you plan to use. [Learn more about API Keys](/docs/advanced/platform/api-keys).
 
+# Providers {% #providers %}
+
+{% table %}
+* Name
+* Description
+---
+* Github actions {% #github-actions %}
+* You can add your CLI to [Github actions](https://github.com/appwrite/setup-for-actions/tree/feat-cli-g2?tab=readme-ov-file#introduction) to automate your Appwrite CLI commands instead of manually using them.
+
+{% /table %}
+
```

Comment:
This seems out of place here. What's the rationale for adding it here?

Reply:
I was thinking of the flow of how the reader would read this documentation, and to me it made sense that this section would proceed the deployment of resources.